Sources/Web3Core/KeystoreManager/BIP32HDNode.swift

```diff
@@ -22,12 +22,11 @@ extension UInt32 {
 }
 
 public class HDNode {
-    public struct HDversion {
-        // swiftlint:disable force_unwrapping
-        public var privatePrefix: Data = Data.fromHex("0x0488ADE4")!
-        public var publicPrefix: Data = Data.fromHex("0x0488B21E")!
-        // swiftlint:enable force_unwrapping
-        public init() {}
+    static var maxIterationIndex = UInt32(1) << 31
+    private struct HDversion {
+        public static var privatePrefix: Data? = Data.fromHex("0x0488ADE4")
+        public static var publicPrefix: Data? = Data.fromHex("0x0488B21E")
+
```

Comment:
Isn't this used anywhere? Otherwise this would be a breaking change and should probably go into `v4.0`

Reply:
Just hanging around here, but I wander what's the point to declare public property within private extension?